Unable to link app as a service

When I try to run :

```
clever service link-app -a my-alias my-app
```

I have an error described below. Despite, my apps are linked to the CLI.

```
[ERROR] Several applications are linked. You can specify one with the "--alias" option. Run "clever applications" to list linked applications.
```

--- a/src/commands/service.js
+++ b/src/commands/service.js
@@ -45,7 +45,7 @@ function list (api, params) {
 }
 
 function linkApp (api, params) {
-  const { alias } = params.options.alias;
+  const { alias } = params.options;
   const [appIdOrName] = params.args;
 
   const s_result = AppConfig.getAppData(alias)

